package.json

```diff
@@ -51,13 +51,14 @@
     "react-dom": "^16.13.1",
     "react-helmet": "^6.1.0",
     "react-measure": "^2.3.0",
-    "react-scripts": "3.4.1",
+    "react-scripts": "3.4.3",
     "react-simple-maps": "^2.1.2",
     "react-sticky": "^6.0.3",
     "semiotic": "^1.20.5",
     "set-order": "^0.3.5",
     "styled-components": "^5.1.1",
     "styled-normalize": "^8.0.7",
+    "terser-webpack-plugin": "^1.4.3",
```

Comment:
is `1.4.5` the minimum version that satisfies this issue? if so I might recommend just making that the base here instead, for extra safety.

Also, since we don't actually use this dependency directly, maybe this would be more appropriate in a [resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/) section instead?